app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -1,6 +1,6 @@
 module UspsInPersonProofing
   class Proofer
-    mattr_reader :token, :token_expires_at
+    AUTH_TOKEN_CACHE_KEY = :usps_auth_token
```

Comment:
```suggestion
    AUTH_TOKEN_CACHE_KEY = 'usps_auth_token'
```

I suspect Redis will need to convert it to a string anyway, so we can save a small conversion I think?

Reply:
Seems a little premature because it's not clear to me there's any performance difference. I expect symbol to string conversion is essentially just the cost of a method call (eg there is an internal representation of a symbol as a string and ruby just needs to return that). I lean towards the slightly more ruby-esque symbol personally. But I'll change it if you want

I'm going to change it to `usps_ippaas_auth_token` as a string or a symbol. Let me know if you want it as a string

Reply:
I don't think it's incredibly impactful and don't feel particularly strongly. Just wanted to flag it as a potential change, fine to leave as-is.